integration_tests/src/main/python/iceberg_test.py

```diff
@@ -37,6 +37,7 @@
 
 rapids_reader_types = ['PERFILE', 'MULTITHREADED', 'COALESCING']
 
+pytestmark = pytest.mark.skip(reason="Skipping all iceberg tests as it's under refactoring: https://github.com/NVIDIA/spark-rapids/issues/12176")
```

Comment:
So if we don't finish fully in 25.04 are we going to revert all of the changes merged in so far or are we going to disable all iceberg support and say that we don't support any of it? To be clear I am fine with either solution as the current iceberg support is partial, old, and not that great to start out with. I just want to make sure we have a plan for what to do if we cannot get everything done.